en/development/errors.rst

```diff
@@ -204,17 +205,18 @@ For more details on HTTP 4xx error status codes see :rfc:`2616#section-10.4`.
 
 For more details on HTTP 5xx error status codes see :rfc:`2616#section-10.5`.
 
+
 You can throw these exceptions from you controllers to indicate failure states,
 or HTTP errors. An example use of the HTTP exceptions could be rendering 404
 pages for items that have not been found::
 
-    public function view($id)
-    {
-        $post = $this->Post->findById($id);
-        if (!$post) {
-            throw new NotFoundException('Could not find that post');
+    public function view($id = null) {
+        $post = $this->Posts->findById($id)->first();
```

Comment:
Also, I don't like how the example appears twice on the page and was and is exactly the same.